lemminx-liberty/src/test/java/io/openliberty/LibertyDiagnosticTest.java

```diff
@@ -59,6 +60,7 @@ public class LibertyDiagnosticTest {
     static String sampleserverXMLURI = new File(srcResourcesDir, "sample-server.xml").toURI().toString();
     static List<WorkspaceFolder> initList = new ArrayList<WorkspaceFolder>();
     LibertyProjectsManager libPM;
+    public static final String MISSING_CONFIGURED_FEATURE_MESSAGE = "This config element does not relate to a feature configured in the featureManager. If the relevant feature is specified in another server configuration file, this message can be ignored. Otherwise, remove this element or add a relevant feature.";
```

Comment:
Why not get this message from the resource bundle instead of hardcoding it here?